optaweb-employee-rostering-backend/src/main/java/org/optaweb/employeerostering/domain/employee/Employee.java

```diff
@@ -65,6 +77,18 @@ public Employee(Integer tenantId, String name, Contract contract,
                     Set<Skill> skillProficiencySet) {
         super(tenantId);
         this.name = name;
+        this.shortId = Arrays.stream(name.split(" ")).limit(3).reduce("", (a,b) -> a + b.charAt(0));
```

Comment:
This deserves extraction into a separate function, one that could be named in an explanatory manner.